packages/database/src/DatabaseType.js

```diff
@@ -164,6 +164,10 @@ export class DatabaseType {
       }
     } else {
       const record = await this.model.create(data);
+
+      /**
+       * @type {string}
+       */
```

Comment:
Without this the type of `id` was `any` which was causing issues. Previously we had to explicitly omit this type from the base type.